packages/rancher-istio-1.10/charts/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.9.5
+appVersion: 1.10.0
 description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/ for details.
 name: rancher-istio
-version: 1.9.5
+version: 1.10.0
```

Comment:
Did upgrade notes show anything new and important or breaking ?

Reply:
Things that were deprecated: 

- values.global.jwtPolicy=first-party-jwt << we do not use this in our base config file so we did not need to remove it.
- values.global.arch << we do not use this in our base config file so we did not need to remove it. 
- Deprecated the `remote` installation profile << we do not point to this profile, ours is the `default` profile

From upgrade notes:
- PILOT_ENABLE_INBOUND_PASSTHROUGH=false  << this was added and is enabled by default. We could add it to our base config, but its something, that if needed could be set via the overlay file. This is something that should be release noted because if you have pods that bind to `localhost` they will no longer be exposed to external pods. You can run `istioctl experimental precheck` to see whats going to be impacted - this something we could consider adding temporarily to our istio-installer image but I am not keen on that idea
- They also changed how the sidecar injector selector works. You can set `values.sidecarInjectorWebhook.useLegacySelectors=true` if you want to continue with the legacy support. Essentially if you can now have a namspace without an istio-injection label, but set the pod annotation to `sidecar.istio.io/inject=true`. So people shouldn't see changes really, but its possible that they took the label off a namespace but left it on a pod, and now that pod would get picked up in a 1.10 world. 

These were the only super noteworthy things I saw when looking through the 1.10 upgrade notes and release annoucements. None of them really required changes to our existing base-config file.